tsconfig.base.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": ["@sapphire/ts-config", "@sapphire/ts-config/extra-strict", "@sapphire/ts-config/decorators", "@sapphire/ts-config/verbatim"],
 	"compilerOptions": {
-		"removeComments": true
+		"removeComments": true,
+		"skipLibCheck": true
```

Comment:
Any reason for this change?

Reply:
error in /decorators where MethodDecorator is somehow no longer generic